routing_model/src/actions.rs

```diff
@@ -106,6 +106,11 @@ impl InnerAction {
         self.our_nodes.push(NodeChange::Online(node.node));
     }
 
+    fn set_offline_state(&mut self, name: &Name) {
+        let node = &mut self.our_current_nodes.get_mut(name).unwrap();
+        self.our_nodes.push(NodeChange::Offline(node.node));
```

Comment:
See `node.is_resource_proofing = false;` in set_online_state.

This is what is actually changing the state, we probably want something like that for offline.
(A new bool may be the simplest for now, but we will likely need to combine some of these into an enum as they are mutually exclusive).

Reply:
self.our_nodes.push(NodeChange::Offline(node.node)); is used only for validating what happened in our tests.